pkg/kubelet/config/common.go

```diff
@@ -29,6 +29,7 @@ import (
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/helper"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
This is because this package does conversion. Could you leave a comment above the import?